scripts/openstack.py

```diff
@@ -296,6 +292,90 @@ def get_parser():
         ],
         conflict_handler='resolve',
         add_help=False,
+        epilog="""test repos:
+
+Test repository can be specified using --test-repo optional argument
+with value in the following formats:  NAME:URL, NAME!PRIORITY:URL
+or @FILENAME. See examples:
+
+1) Essential usage requires to provide repo name and url:
+
+    --test-repo foo:http://example.com/repo/foo
+
+2) Repo can be prioritized by adding a number after '!' symbol
+   in the name:
+
+    --test-repo bar!10:http://example.com/repo/bar
+
+3) Repo data can be taken from a file by simply adding '@' symbol
+   at the beginning argument value, for example from yaml:
+
+    --test-repo @path/to/foo.yaml
```

Comment:
Should this be `--test-repo '@path/to/foo.yaml'` ?

Reply:
There should be no difference `'@path/to/foo.yaml'` or `@path/to/foo.yaml`.